Frontend/src/components/StudentPage/StudTakeAssessPage.js

```diff
@@ -61,20 +86,27 @@ export default function StudentTakeAssessPage() {
                     data.map(item=> quizlist.push(item._id)) //all quiz ids!!
                     setQuizlist(quizlist);
                     //console.log("check qidlist state " + qlist);
-                    if(data == null) {console.log(Error)}
+                    
+                    // if(data == null || data.length == 0) {
+                    //     console.log(Error);
+                    // }
+                    // else{
+                    //     console.log(data);
+                    // }
+
                 })
         }
     });
+    */
     //qlist is an array of quiz object ids by coach
     //console.log(qlist[0]);
     //console.log("check the" + qlist);
     
     //{<QuizzesList data={(currquizzes)}/>}
+    //{<QuizzesList data={(quizlist)}/>}
```

Comment:
Could we take all of this commented out code and put it into a google doc? I just made a folder in the drive for storing old code so that we have it. It's cluttering up the file a ton, and there's honestly a pretty low chance of using it anyways